apps/server/src/api/admin/tasks/get.ts

```diff
@@ -1,5 +1,15 @@
 import { middyfy } from '../../../helpers/wrapper';
-import { $Tasks } from '../../../schemas';
+import { $Tasks, Task } from '../../../schemas';
 import tasks from '../../../tasks';
 
-export const main = middyfy(null, $Tasks, true, async () => tasks.map((t) => ({ id: t.id, name: t.name })));
+export const main = middyfy(null, $Tasks, true, async (event) => {
+  const overlap = (a: string[], b: string[]): boolean => a.some((v) => b.includes(v));
+  const listOfAllTasks = tasks.map((t) => ({ id: t.id, name: t.name, groups: t.groups }));
+  const listOfTasks: Task[] = [];
+  listOfAllTasks.forEach((task) => {
```

Comment:
consider: I think you can probably achieve this more succicntly with [`.filter()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)

e.g.

```ts
return listOfAllTasks.filter((task) => overlap(event.auth.payload.groups, task.groups))
```